app/_includes/md/kic/support.md

```diff
@@ -15,7 +15,7 @@
   <tr>
     <td>1.29</td>
     <td>N/A</td>
-    <td>Feb 2022</td>
+    <td>Dec 2024</td>
```

Comment:
```suggestion
    <td>Feb 2025</td>
```

This is K8s 1.29 EOL, isn't it? According to the docs, it is `2025-02-28`
![image](https://github.com/Kong/docs.konghq.com/assets/9593424/398f7379-9065-473c-b8c2-4315110670e7)
[src](https://kubernetes.io/releases/#release-v1-29)

Reply:
I took a look once again and it was me who made the mistake. For all the previous K8s versions we were using the EOL date and here it should be the same. 👍